vars/terraform.groovy

```diff
@@ -107,20 +108,19 @@ def call(userConfig = [:]) {
 
             if (isBuildOnChangeRequest) {
               stage('🗣 Notify User on the PR') {
-                final String planFileUrl = "${env.BUILD_URL}artifact/terraform-plan-for-humans.txt"
+                final String planFileUrl = "${env.BUILD_URL}artifact/${planFileName}"
                 publishChecks name: "terraform-plan",
                   title: "Terraform plan for this change request",
                   text: "The terraform plan for this change request can be found at <${planFileUrl}>.",
                   detailsURL: planFileUrl
               }
               stage('💸 Report estimated costs') {
-                try {
+                withCredentials([string(credentialsId: 'infracost-api-key', variable: 'INFRACOST_API_KEY')]) {
```

Comment:
Could you work around adding back the `try{}` block again please?
Anything going wrong in the infracost process should not impact our ability to execute terraform steps